Formula/kibana-full.rb

```diff
@@ -23,7 +23,13 @@ def install
       "x-pack",
     )
 
-    cd prefix do
+    Pathname.glob(libexec/"bin/*") do |f|
+      next if f.directory?
+      bin.install libexec/"bin"/f
+    end
+    bin.env_script_all_files(libexec/"bin", { "KIBANA_PATH_CONF" => etc/"kibana", "DATA_PATH" => var/"lib/kibana/data" })
```

Comment:
DATA_PATH is deprecated/replaced with kibana.yml path.data in 7.2+ but shouldn't have any adverse effects

Reply:
https://github.com/elastic/kibana/pull/32049

Reply:
Sorry, I've been meaning to open a bug for this. I think that `kibana-keystore` does not support `path.data` in `kibana.yml`, it reads the data path directly. Thus the need to use `DATA_PATH` here.

I opened elastic/kibana#43539.